TeamCode/src/main/java/org/firstinspires/ftc/teamcode/servoClaw.java

```diff
@@ -9,14 +9,9 @@
 public class servoClaw {
     public Servo servoClaw;
 
-    public void runServoClaw(boolean openClaw, boolean closeClaw) {
-
-        if (openClaw){
-            servoClaw.setPosition(clawOpenValue); //open
-        }
-        else if(closeClaw){
-            servoClaw.setPosition(clawCloseValue); //closed
-        }
+    public void runServoClaw(double clawInput) {
```

Comment:
Change the name of the class to something more like setClawServoPosition or something along those lines as it explains the purpose of the code alot quicker